test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -83,5 +85,34 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
         }
+
+        [Theory, TraceTest]
```

Comment:
That's something I added previously. If you enable `DNX_TEST_TRACE` it will print out `Test Information: Begin test/End test: <test>`. This is sort of redundant now given we use the `-verbose` flag for xunit which prints out the beginning of the test